sys/tracefd/include/tracefd.h

```diff
@@ -45,12 +45,22 @@ extern "C" {
 
 /**
  * @def TRACEFD_NUMOF
- * @brief Number of trace file descriptors
+ * @brief Number of trace file descriptors specific to the application
  */
 #ifndef TRACEFD_NUMOF
-#  define TRACEFD_NUMOF 1
+#  define TRACEFD_NUMOF 0
```

Comment:
why changing to 0 by default ?

Reply:
Because the module can be used without file, just with stderr and stdout descriptors which are always available.